src/material-experimental/mdc-slider/slider.ts

```diff
@@ -298,6 +312,18 @@ export class MatSliderThumb implements AfterViewInit {
   /** Event emitted every time the MatSliderThumb is focused. */
   @Output() readonly _focus: EventEmitter<void> = new EventEmitter<void>();
 
+  /**
+   * A callback function that is called when the
+   * control's value changes in the UI (ControlValueAccessor).
+   */
+  _onChange: (value: any) => void = () => {};
+
+  /**
+   * A callback function that is called by the forms API on
+   * initialization to update the form model on blur (ControlValueAccessor).
+   */
+  private _onTouched: () => any = () => {};
```

Comment:
I think that the correct type is `() => void`.

Reply:
Seems like we have this wrong in a few places. I copied that line from `src/material/slider/slider.ts`